star-rating/README.md

```diff
@@ -1,13 +1,68 @@
-# Star Rating Field-Type
+<p style="text-align: center">
+  <a href="https://www.storyblok.com/" rel="noopener">
+ <img width=150px src="https://a.storyblok.com/f/51376/3856x824/fea44d52a9/colored-full.png" alt="Storyblok" style="display:inline-block;margin:20px;"></a>
+</p>
 
-This field-type displays a star rating.
+<h1>
+<img src="assets/icon.svg" height="26px">
+Star Rating Field Type
+</h1>
 
-Name | Description | Author
------------- | ------------- | -------------
-Star Rating | A field-type to display a star rating | [Lisi Linhart](https://github.com/lisilinhart/)
+A field-type to display a star rating.
 
+![screenshot.gif](docs/screenshot.png)
 
-## Amount of stars
+![demo.gif](docs/demo.gif)
 
-The default amount of stars is 5.
-You can change the amount of stars by passing an option with the name `amount` and the value should be the amount of stars you want.
\ No newline at end of file
+## How to use
+
+To add this field type to a Storyblok space, you need to
+
+1. Rename the .env.local.example file to .env.local
+2. Open the .env.local file and update the `STORYBLOK_PERSONAL_ACCESS_TOKEN` (can be found here: [management api token](https://app.storyblok.com/#/me/account?tab=token))
+
+```bash
+STORYBLOK_PERSONAL_ACCESS_TOKEN=[your management api token here]
+```
+
+3. From the project's root, run:
+
+```bash
+# install dependencies
+yarn install
+
+# deploy the plugin using our @storyblok/field-plugin-cli
+yarn deploy
+```
+
+4. `yarn deploy` will ask you for the plugin's name and also the place this plugin should be added (My Plugins or Partner Portal). Please provide them in order to continue.
+
+5. Once the deploy is done successfully, you can continue its usage following the "Set up" section right below.
+
+Ps: If you are facing some issue performing these steps, you could also check our updated article on how to create and deploy field-plugins here: [Introduction to field-plugin](https://www.storyblok.com/docs/plugins/introduction).
+
+# App Description
+
+Star Rating component that can be used in a variety of scenarios, such as:
+
+- Rating movie
+- Rating some place
+- Rating some musician performance
+
+## How to set up
```

Comment:
This section is vague. I didn't realize this section was about configuration on the storyblok.com side. We could have a small screenshot to help them better understand it.

Reply:
I meant a new screenshot where you show them how to add an option.